cmd/gpu_fakedev/gpu_fakedev.go

```diff
@@ -148,6 +149,46 @@ func addSysfsDriTree(root string, opts *genOptions, i int) error {
 	return nil
 }
 
+func addBusFiles(root string, opts *genOptions, i int) error {
+	pciName := fmt.Sprintf("0000:00:0%d.0", i)
+	base := filepath.Join(root, "bus/pci/drivers/i915", pciName)
```

Comment:
`filepath` module is intended to hide difference between Windows and Linux path separators.  That's not really relevant when handling Linux sysfs paths.   I.e. you can do `sed -i 's/filepath\./path./g' gpu_fakedev.go` for this PR.

Reply:
Filepath is a great portable way to join paths. What is the gain in functionality we achieve by avoiding it?

Reply:
Both work, but `path.Join()` is _consistent with existing code_ using `Sprintf()`s & format strings having '/' as their path separators.  I'm not expecting you to change those existing `Sprintf()`s to do portable path handling [1], so next best thing is using `path.Join()`.

[1] Unless you think that running DRA under Windows with GPU faking is something that would need to be supported?  That will need similar changes also on DRA side, as it's also using `Sprintf()`s with format strings having '/' path separators.

Reply:
Yes indeed, I didn't particularly fancy the original way of doing it, hence the inconsistency.

Reply:
Actually both filepath.Join and path.Join are used mixed inside the device plugins repo. IMO the better option is filepath, even if the existing "c-coder was here" sprintf-usage leans more towards the path.Join alternative.